cookbooks/Function-Calling-OpenAI/typescript/README.md

```diff
@@ -0,0 +1,21 @@
+# Setup
```

Comment:
Should we seperate into two subdirs for python and ts? It might be a bit confusing to have both in the same dir